Tools/LyTestTools/tests/integ/sanity_tests.py

```diff
@@ -15,40 +15,51 @@
 import ly_test_tools.environment.process_utils as process_utils
 import ly_test_tools.environment.waiter as waiter
 
-pytestmark = pytest.mark.SUITE_smoke
```

Comment:
I assume these are registered without filtering? These tests need to stay live and running on every AR execution

Reply:
yes these are explicitly registered as smoke already